app/constants/calls.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+const RefreshConfigMillis = 20 * 60 * 1000; // Refresh config after 20 minutes
+
+const RequiredServer = {
+    FULL_VERSION: '6.3.0',
+    MAJOR_VERSION: 6,
+    MIN_VERSION: 3,
+    PATCH_VERSION: 0,
+};
+
+const PluginId = 'com.mattermost.calls';
```

Comment:
Would this PluginId be suject to change ? If so, perhaps we need to have this documented in the `README.md` file as how to enable/set-up calls in a project.

Reply:
We are planning on changing the `PluginId` in the future, but it's not something that the user would be able to do (or needs to be aware of), so I don't think we need to add anything to the `README.md`.

As for enable/set-up calls, we have Calls-specific docs for that, I don't think it's needed in the mobile docs.

Reply:
How will changing the id in the future will impact previous installations and such?

Reply:
Yep, we're going to keep that in mind and make sure mobile doesn't break.